### PR TITLE
Add docker-compose for dummy model

### PR DESCRIPTION
**Pull request by KonstantinKorotaev**

### CODE REVIEW

File: label_studio_ml/examples/dummy_model/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+```bash
+$ curl http://localhost:9090/health
+{"status":"UP"}
+```
+
+Then connect running backend to Label Studio:
+
+```bash
+label-studio start --init new_project --ml-backends http://localhost:9090 --template image_classification
```
Comment (niklub) [inline label_studio_ml/examples/dummy_model/README.md:19]: this is something outdated

File: label_studio_ml/examples/dummy_model/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+
+#### Inference module
+In order to create module for inference, you have to declare the following class:
+
+```python
+from htx.base_model import BaseModel
+
+# use BaseModel inheritance provided by pyheartex SDK 
+class MyModel(BaseModel):
```
Comment (niklub) [inline label_studio_ml/examples/dummy_model/README.md:54]: this too, pyheartex, BaseModel, etc.

**Review by niklub — CHANGES_REQUESTED** [review]

File: label_studio_ml/examples/dummy_model/README.md (hunk 2 above)
Comment (KonstantinKorotaev) [reply label_studio_ml/examples/dummy_model/README.md]: Fixed

File: label_studio_ml/examples/dummy_model/README.md (hunk 1 above)
Comment (KonstantinKorotaev) [reply label_studio_ml/examples/dummy_model/README.md]: Fixed

**Review by makseq — APPROVED** [review]

**Pull request merged**